dependency-check-suppressions.xml

```diff
@@ -106,32 +106,39 @@
         <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.dataformat/jackson-dataformat-cbor@.*$</packageUrl>
         <cve>CVE-2020-28491</cve>
     </suppress>
-    <suppress until ="2022-10-01">
+    <suppress until="2022-10-01">
         <notes><![CDATA[
                 spring-security-crypto-5.7.2.jar
         ]]></notes>
         <packageUrl regex="true">^pkg:maven/org.springframework.security/spring-security-crypto@.*$</packageUrl>
         <cve>CVE-2020-5408</cve>
     </suppress>
-    <suppress until ="2022-10-01">
+    <suppress until="2022-10-01">
         <notes><![CDATA[
                 spring-core-5.3.22.jar
         ]]></notes>
         <packageUrl regex="true">^pkg:maven/org.springframework/spring-core@.*$</packageUrl>
         <cve>CVE-2016-1000027</cve>
     </suppress>
-    <suppress until ="2022-10-01">
+    <suppress until="2022-10-01">
         <notes><![CDATA[
                 spring-web-5.3.22.jar
         ]]></notes>
         <packageUrl regex="true">^pkg:maven/org.springframework/spring-web@.*$</packageUrl>
         <cve>CVE-2016-1000027</cve>
     </suppress>
-    <suppress until ="2022-10-01">
+    <suppress until="2022-10-01">
         <notes><![CDATA[
                 spring-security-crypto-5.4.10.jar, spring-security-web-5.4.10.jar
         ]]></notes>
         <packageUrl regex="true">^pkg:maven/org.springframework.security/spring-security.*$</packageUrl>
         <cve>CVE-2022-22978</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+                snakeyaml-1.26.jar
+        ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org.yaml/snakeyaml.*$</packageUrl>
+        <cve>CVE-2022-25857</cve>
+    </suppress>
```

Comment:
Can you add a date for expiry?
Also you can upgrade this to version 1.31

Reply:
The dependency was upgraded in another PR:
https://github.com/hmcts/sscs-tribunals-case-api/pull/2925

This branch will be rebased or the update will be merged accordingly.

Reply:
merged with Del changes so it will be updated to 1.31